core/src/main/scala/kafka/controller/KafkaController.scala

```diff
@@ -1472,6 +1525,49 @@ class KafkaController(val config: KafkaConfig, zkUtils: ZkUtils, val brokerState
     }
   }
 
+  case class LeaderAndIsrResponseReceived(LeaderAndIsrResponseObj: AbstractResponse, brokerId: Int) extends ControllerEvent {
```

Comment:
I'd prefer renaming `TopicDeletionStopReplicaResult` to `TopicDeletionStopReplicaResponseReceived`.